qgreenland/config/layers/Internet-required data/__settings__.py

```diff
@@ -6,6 +6,7 @@
         LayerIdentifier("image_mosaic_2015"),
         LayerIdentifier("sdfi_satellite_orthophotos"),
         LayerIdentifier("sdfi_topo_map"),
+        LayerIdentifier("geus_geological_map"),
```

Comment:
Is it worth breaking online-only layers up into different categories at this point?

It would be nice to have this layer easily discoverable in the `Geology` group...maybe we could distinguish online-only layers in some other way? Something to think about...

Reply:
I think that's worthwhile, definitely. I think warrants an issue and a dedicated discussion.

Reply:
https://github.com/nsidc/qgreenland/issues/727